examples/chart/teleport-cluster/templates/auth/pvc.yaml

```
@@ -0,0 +1,21 @@
+{{- $auth := mustMergeOverwrite .Values .Values.auth -}}
+{{- if $auth.persistence.enabled }}
+  {{/* Disable persistence for aws and gpc modes */}}
```

```suggestion
  {{/* Disable persistence for aws and gcp modes */}}
```

As an aside, I think if we (eventually) end up adding an `azure` chartMode then we should rework all these conditions into some kind of `cloudProvider` "meta" check (via template?) so we don't have to enumerate each individual `chartMode` at multiple different locations in the chart.

---

Before we were segregating "stanadalone" and "custom", now it's an implicit category "cloud providers happy path without persistence". I guess it makes sense to at least explicit this, maybe to have a" no persistence mode" list or something.